### DISCUSSION TITLE
Your ideas for a new name

### DISCUSSION THREAD

**Discussion by loleg** [dribdat/design#13] [Q&A]
Starting the new year with a fresh take, I would like to think about changing the name of the project. I still like "dribdat" personally, but I see it misspelled constantly (_dripdat, dribat, dribthat, dribdee_ ...), and some people have trouble with pronounciation (drih'b'dat is not a word that just rolls off your tongue, and might even be mistaken for [Klingon](https://en.m.wikipedia.org/wiki/Klingon_language) 😉). I've already worked in some variations to the name, like Dribs, DRIBD, dridbot, into the project, but I am not sure this nomenclature will stick.

What do y'all think about that? Any suggestions?

**Comment by kinlearn** [comment]
Hi @loleg,
I appreciate your thoughtful consideration for a new project name. Personally, I also like the original name "Dribdat." However, I understand the need for a change due to spelling challenges.
After some reflection, I'd like to suggest a few names:
### DribSync:
**Reason:** Conveys a sense of synchronization and collaboration, aligning with the project's core principles. It's modern, catchy, and easy to remember.

### DribCoLab or CollabDrib or CollaDrib:

**Reason:**  Blends "collaboration" with "Drib," emphasizing the essence of collaborative work. 

These options aim to maintain the spirit of the original name while addressing concerns about spelling and pronunciation. Let me know your thoughts!

**Reply by kinlearn** [reply]
Your suggestion "Dribs" is also cool! The only confusion it might have is some people might mistake it for dribbs (two B's) and vice versa. Dribs feels more modular and hackable. A couple of ideas - 
### Comparing Potential Names for Dribdat

| Name | Pros | Cons |
|---|---|---|
| Dribs | Short, catchy, playful | Lacks descriptive element, can be misspelled with dribbs |
| Dribbler | Creative, playful, suggestive of action | resembles too much with Dribbble |
| DribNest | Unique, evokes community, collaboration and coziness  | May not be clear on purpose |
| DribHub | Descriptive, straightforward, suggests central resource | Lacks playfulness, sounds too generic |

These names were thought on basis of availability, familiarity, ease of pronunciation, staying relevant with original purpose, etc. I am constantly thinking about it. Looking forward for your feedback. Thanks. 😊

**Comment by stefnnn** [comment]
Great idea @loleg. While I do find «dribdat» a very likeable name, I never could associate it with anything, hence I had a hard time to remember it. How important is the «data» part nowadays for the project? If the focus is more on general hack and innovation events then here are a two suggestions:

- JamLog
- InnoQuest
- Hackfest
- «Gether»

Let me know if you need help with a logo once you settled on a name ;-)

**Reply by loleg** [reply]
Thanks for the ideas! What do you folks think of [EveryHack](https://everyhack.day)?

**Reply by stefnnn** [reply]
Yes, I like!

**Comment by kinlearn** [comment]
Yeah, sounds good to me. Thankfully the brandname is very untouched despite of being so chatchy..

**Comment by loleg** [comment]
We have another suggestion from Lukas A.: `Hack10X`
